perf.groovy

```diff
@@ -309,8 +308,8 @@ def static getFullPerfJobName(def project, def os, def isPR) {
 
         // Save machinedata.json to /artifact/bin/ Jenkins dir
         def archiveSettings = new ArchivalSettings()
-        archiveSettings.addFiles('Perf-*.log')
-        archiveSettings.addFiles('Perf-*.xml')
+        archiveSettings.addFiles('${outputDirectory}/Perf-*.log')
+        archiveSettings.addFiles('${outputDirectory}/Perf-*.xml')
         archiveSettings.addFiles('machinedata.json')
```

Comment:
No, this is the new location to get the output files to be archive (used to be generated in sandbox and copied to the CORECLR_ROOT).